### PR TITLE
[T4A2][T16-B1] Ong Li-Jen

### PR DESCRIPTION
**Pull request by lijen1994**
Changed Parser.java and Person.java to better reflect class-level members.

### CODE REVIEW

**Comment by lijen1994** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -24,14 +26,16 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
...
+        this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arglist
+        this.sequenceNumber = nextSequenceNumber++;
     }
 
     /**
      * Copy constructor.
      */
     public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
+        this.sequenceNumber = nextSequenceNumber++;
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/Person.java:38]: Good job on updating the copy constructor as well.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -96,7 +96,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args){
+    private static Command prepareAdd(String args){
```
Comment (PierceAndy) [inline src/seedu/addressbook/parser/Parser.java:99]: Good job on making these private methods static as well. As `parseCommand` was made static, any methods that it depends on has to be made static as well.

**Comment by PierceAndy** [comment]
Commit messages should be descriptive, such that others viewing your code can understand what you're trying to achieve in a commit. Try taking a look at your peer's commit messages to get an idea of what makes a good or bad commit message.

@lijen1994 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**